doc/source/dev/reviewer_guidelines.rst

```diff
@@ -0,0 +1,119 @@
+.. _reviewer-guidelines:
+
+===================
+Reviewer Guidelines
+===================
+
+Reviewing open pull requests (PRs) helps move the project forward. We encourage
+people outside the project to get involved as well; it's a great way to get
+familiar with the codebase.
+
+Who can be a reviewer?
+======================
+
+Reviews can come from outside the NumPy team -- we welcome contributions from
+domain experts (for instance, `linalg` or `fft`) or maintainers of other
+projects. You do not need to be a NumPy maintainer (a NumPy team member with
+permission to merge a PR) to review.
+
+If we do not know you yet, consider introducing yourself in `the mailing list or
+Slack <https://numpy.org/community/>`_ before you start reviewing pull requests.
+
+Communication Guidelines
+========================
+
+- Every PR, good or bad, is an act of generosity. Opening with a positive
+  comment will help the author feel rewarded, and your subsequent remarks may be
+  heard more clearly. You may feel good also.
+- Begin if possible with the large issues, so the author knows they've been
+  understood. Resist the temptation to immediately go line by line, or to open
+  with small pervasive issues.
+- You are the face of the project, and NumPy some time ago decided `the kind of
+  project it will be <https://numpy.org/code-of-conduct/>`_: open, empathetic,
+  welcoming, friendly and patient. Be `kind
+  <https://youtu.be/tzFWz5fiVKU?t=49m30s>`_ to contributors.
+- Do not let perfect be the enemy of the good, particularly for documentation.
+  If you find yourself making many small suggestions, or being too nitpicky on
+  style or grammar, consider merging the current PR when all important concerns
+  are addressed. Then, either push a commit directly (if you are a maintainer)
+  or open a follow-up PR yourself.
+- If you need help writing replies in reviews, check out some `Standard replies
+  for reviewing
+  <https://scikit-learn.org/stable/developers/tips.html#saved-replies>`_.
+
+Reviewer Checklist
+==================
+
+- Is the intended behavior clear under all conditions? Some things to watch:
+   - What happens with unexpected inputs like empty arrays or nan/inf values?
+   - Are axis or shape arguments tested to be `int` or `tuples`?
+   - Are unusual `dtypes` tested if a function supports those?
+- Should variable names be improved for clarity or consistency?
+- Should comments be added, or rather removed as unhelpful or extraneous?
+- Does the documentation follow the :ref:`NumPy guidelines<howto-document>`? Are
```

Comment:
still an error in the reference here, needs a space

Reply:
or maybe this does work, since you're doing it in multiple place. let me go look at the html